### PR TITLE
Upgrade to Ubuntu 20 for Circle [BT-576]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: servers/dsub/jobs/test/base_test_cases.py [hunk 1]
```diff
@@ -25,18 +25,21 @@
 from jobs.models.aggregation_response import AggregationResponse
 from jobs.models.status_counts import StatusCounts
 
-DOCKER_IMAGE = 'ubuntu:14.04'
+DOCKER_IMAGE = 'ubuntu:20.04'
```
Comment (breilly2) [inline servers/dsub/jobs/test/base_test_cases.py:28]: Curious to know if these tests work. However, according to the README:
> dsub support is deprecated and will be removed in a future release.

File: .circleci/config.yml [hunk 2]
```diff
@@ -36,10 +37,32 @@ jobs:
           key: node-modules-{{ .Branch }}--{{ checksum "ui/package.json" }}
       - run:
           name: nvm install node and UI dependencies
+          # The Circle Ubuntu machine images of yore apparently came with nvm installed, but the current images require
```
Comment (breilly2) [inline .circleci/config.yml:40]: Looks like this all worked in the CircleCI build for this PR.

**Review by breilly2 — APPROVED** [review]
👍 Thanks for tackling this!

**Review by jgainerdewar — APPROVED** [review]

File: servers/dsub/jobs/test/base_test_cases.py (hunk 1 above)
Comment (mcovarr) [reply servers/dsub/jobs/test/base_test_cases.py:28]: it looks like the dsub tests were actually [disabled last year](https://github.com/DataBiosphere/job-manager/pull/726/files)

**Pull request merged**